packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1224,6 +1254,12 @@ class TerminalHandler {
           return true;
         }
         return false;
+      case 'M':
+        if (residentRunner.supportsWriteSkSl) {
```

Comment:
Skia's name convention is `SkSL` with capitalized `SL` (stands for Shader Language).